tasks/gov.js

```diff
@@ -47,21 +47,230 @@ task("gov:setPricingConfig")
+
+        console.log("\nNEW pricing config:");
+        console.log(et.dumpObj(curr));
+        console.log("\n");
+    });
+
+task("gov:forkSetChainlinkPriceFeed", "Impersonate governor admin on mainnet fork and set chainlink price feed for an asset")
+    .addPositionalParam("underlying", "underlying mainnet asset address")
+    .addPositionalParam("chainlinkAggregator")
+    .setAction(async ({ underlying, chainlinkAggregator }) => {
+        const {et, ctx, admin} = await setupGovernanceFork("governor");
+
+        await et.taskUtils.runTx(ctx.contracts.governance.connect(admin).setChainlinkPriceFeed(underlying, chainlinkAggregator, await ctx.txOpts()));
+    });
+
+task("gov:forkSetAssetConfig", "Impersonate governor admin and run governance actions against governance contract")
+    .addPositionalParam("underlying", "underlying mainnet asset address")
+    .addOptionalParam("isolated")
+    .addOptionalParam("cfactor")
+    .addOptionalParam("bfactor")
+    .addOptionalParam("twap")
+    .setAction(async ({ underlying, isolated, cfactor, bfactor, twap }) => {
+        const {et, ctx, admin} = await setupGovernanceFork("governor");
+
+        let curr = await ctx.contracts.markets.underlyingToAssetConfig(underlying);
+
+        console.log("Current asset config:");
+        console.log(et.dumpObj(curr));
+        console.log("\n\n");
+
+        let updated = {};
+
+        updated.eTokenAddress = curr.eTokenAddress;
+        updated.borrowIsolated = isolated === undefined ? curr.borrowIsolated : parseBool(isolated);
+        updated.collateralFactor = cfactor === undefined ? curr.collateralFactor : parseFactor(cfactor);
+        updated.borrowFactor = bfactor === undefined ? curr.borrowFactor : parseFactor(bfactor);
+        updated.twapWindow = twap === undefined ? curr.twapWindow : parseTwap(twap);
+
+        console.log("NEW asset config:");
+        console.log(et.dumpObj(updated));
+        console.log("\n\n");
+
+        await et.taskUtils.runTx(ctx.contracts.governance.connect(admin).setAssetConfig(underlying, updated, await ctx.txOpts()));
+    });
+
+task("gov:forkHealthScoreDiff", "Compare the health scores of accounts from a pair of JSON files")
+    .addPositionalParam("preGovActionFileName", "file name without .json suffix")
+    .addPositionalParam("postGovActionFileName", "file name without .json suffix")
+    .addPositionalParam("removeFilesAfterParsing", "true or false")
+    .setAction(async ({preGovActionFileName, postGovActionFileName, removeFilesAfterParsing}) => {
+        const fs = require('fs');
+        
+        const prePath = `${preGovActionFileName}.json`;
+        const postPath = `${postGovActionFileName}.json`;
+        
+        try {
+            const pre_gov_scores = require(`../${prePath}`);
+            const post_gov_scores = require(`../${postPath}`);
+
+            for (let account of Object.keys(pre_gov_scores)) {
+                if (
```

Comment:
rather than only check whether the account is in violation after performed governance action, it'd be better to check how much given account was affected. what if the health score after performed action is 1.01? technically it wouldn't be violation, but the account would be at serious risk of liquidation. would be great to know about it.

hence, I recommend to slightly rework the logic to:
- catch the accounts for which the health score dropped to risky levels (< 1.25 ?)
- ignore the accounts for which health score is at risky levels but hasn't changed pre- vs post-action
- store the account size (collateral and liabilities value) in `gov:forkAccountsAndHealthScores` and display it here with the message. affected accounts size has impact on decision making

Reply:
@kasperpawlowski good idea. Will fix these today.